pyproject.toml

```diff
@@ -0,0 +1,7 @@
+[project]
+name = "embryo-ode"
```

Comment:
```suggestion
name = "epipeODE"
```

Reply:
If we are to have this as a package the name should be very specific as we are only modeling one step of embryogenesis: hence epipeODE.